src/UI/Dropdown.elm

```diff
@@ -391,6 +391,8 @@ selectAttrs (Dropdown prop _) =
 listAttrs : List (Attribute msg)
 listAttrs =
     [ Element.width Element.fill
+    , Element.height <| px 200
```

Comment:
This is not limiting it, it's forcing a fixed value :eyes: 
Think we can manage to make it `shrink |> maximum XXX` ?

what about having a `withMaximumListHeight : Int -> ...`?